solr/core/src/test/org/apache/solr/handler/admin/CoreAdminOperationTest.java

```diff
@@ -167,23 +167,29 @@ public void testRenameUnexpectedFailuresResultIn500SolrException() {
 
   @Test
   public void testRenameMissingCoreParamResultsIn400SolrException() {
+    final var coreAdminHandler = mock(CoreAdminHandler.class);
+    final var callInfo1 = new CoreAdminHandler.CallInfo(coreAdminHandler, mockRequest, null, null);
+
```

Comment:
Earlier the code exists at the start If all the required parameters are not passed. 
Now that validation is removed from here and CoreAdminOperation relies on RenameCore class for it,  test started failing as passed mock does not contain the Handler object. That's why I created new mock and passed it. Also Let me know If you want to change the variable name.

Reply:
Thanks for calling these out.

> That's why I created new mock and passed it

The mock makes sense IMO 👍 

You likely already know this, but I'll mention it as an FYI, since it's a common gotcha in writing Solr tests: any tests that use mocks need to have a `assumeWorkingMockito` call in a `@BeforeClass` method.  It looks like we've already got in this class, so no need to change anything here!  Just wanted to put the FYI out there.

> Let me know If you want to change the variable name.

Haha, 'callInfo1' isn't the best name, maybe.  I probably would've chosen something like `renameCallInfo` or 'callInfoWithMockHandler`.  (That latter option makes it clear why we're not using the 'callInfo' like all the other tests.). But the test methods are small and clear, so it's not a big deal, ultimately.  So it's up to you.